src/sentry/tasks/weekly_reports.py

```diff
@@ -279,7 +279,7 @@ def zerofill_data(data):
         orderby=[OrderBy(Column("time"), Direction.ASC)],
     )
     if features.has("organizations:weekly-report-logs", ctx.organization):  # TODO(isabella): remove
-        query.set_limit(10000)
+        query = query.set_limit(10000)
```

Comment:
Is there a test for this feature that could verify that the query was set?

Reply:
No worries! Just wanted to make sure there is good coverage. Also just noting to myself to make tests to check that important values are set properly since I just had a similar issue with a Discord PR. 🫣